parsimonious/grammar.py

```diff
@@ -14,8 +14,10 @@
     expression)
 from parsimonious.nodes import NodeVisitor
 from parsimonious.utils import StrAndRepr, evaluate_string
+from six import text_type, iterkeys, itervalues, iteritems, \
```

Comment:
As per PEP 8 and in consistency with the rest of the project, let's use parens to do the line wrapping.